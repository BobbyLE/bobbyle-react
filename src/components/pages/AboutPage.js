import React from 'react';
import GradientTitle from '../front/GradientTitle';


const AboutPage = () => {
  const beginColor = "#009cf3";
  const endColor = "#16d6d9";
  return (
    <div className="container main about">
      <section className="main__header column">
        <GradientTitle 
        beginColor={beginColor}
        endColor={endColor}>
          About
        </GradientTitle>
      </section>
      <section className="presentation column small-12">
        <aside className="presentation__photo">
        </aside>
        <section className="presentation__text">
          <h2 className="presentation__text__title">Bobby Le</h2>
          <span className="presentation__text__subtitle">Web Developer, Photographer</span>
          <h3>Presentation</h3>
          <p></p>
        </section>
      </section>
      <section className="experiences column small-12 medium-6">
        <h3>Experiences</h3>
      </section>
      <section className="skills column small-12 medium-6">
        <h3>Skills</h3>
      </section>
    </div>   
  )
};

export default AboutPage;