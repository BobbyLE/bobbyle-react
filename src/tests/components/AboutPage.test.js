import React from 'react';
import { shallow } from 'enzyme';
import AboutPage from '../../components/pages/AboutPage';

test('should correctly render AboutPage', () => {
  const wrapper = shallow(<AboutPage />);
  expect(wrapper).toMatchSnapshot();
});
