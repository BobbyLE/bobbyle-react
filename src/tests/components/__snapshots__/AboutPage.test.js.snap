// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render AboutPage 1`] = `
<div
  className="container main about"
>
  <section
    className="main__header column"
  >
    <GradientTitle
      beginColor="#009cf3"
      endColor="#16d6d9"
    >
      About
    </GradientTitle>
  </section>
  <section
    className="presentation column small-12"
  >
    <aside
      className="presentation__photo"
    />
    <section
      className="presentation__text"
    >
      <h2
        className="presentation__text__title"
      >
        Bobby Le
      </h2>
      <span
        className="presentation__text__subtitle"
      >
        Web Developer, Photographer
      </span>
      <h3>
        Presentation
      </h3>
      <p />
    </section>
  </section>
  <section
    className="experiences column small-12 medium-6"
  >
    <h3>
      Experiences
    </h3>
  </section>
  <section
    className="skills column small-12 medium-6"
  >
    <h3>
      Skills
    </h3>
  </section>
</div>
`;
